.github/workflows/release.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         go-version: '=1.21.8'
     - name: Update manifests
       run: |
-        make release RELEASE_TAG=${{ env.TAG }}
+        make release RELEASE_TAG=${{ env.TAG }} REGISTRY=${{ env.GHCR_REGISTRY }}
```

Comment:
NOTE: I am setting this to ghcr and not the prime registry to keep the previous behaviour. If not set explicitly here like we had before, it will use `PRIME-REGISTRY-REGISTRY` for `REGISTRY` since it is the recent value  set to the `REGISTRY` env var